mesonbuild/backend/vs2010backend.py

```diff
@@ -1159,7 +1159,8 @@ def gen_vcxproj(self, target, ofname, guid):
         # Size-preserving flags
         if '/Os' in o_flags:
             ET.SubElement(clconf, 'FavorSizeOrSpeed').text = 'Size'
-        else:
+        # Note: setting FavorSizeOrSpeed with clang-cl conflicts with /Od and can make debugging difficult, so don't.
+        elif not '/Od' in o_flags:
```

Comment:
CI fails here because pylint dislikes the coding style.

```suggestion
        elif '/Od' not in o_flags:
```

Reply:
Cool, done.

Thanks